src/components/error-page/ErrorPage.jsx

```diff
@@ -32,19 +33,21 @@ const ErrorPage = ({
         </Col>
       </Container>
     </main>
-    <SiteFooter />
+    {showSiteFooter && <SiteFooter />}
```

Comment:
In retrospect, it may be better for us to add a new component that works without needing any of the stuff around logged in state, for now I managed to get this to work by checking various pieces of state exist or don't. Not as clean but works